browser/scripts/ui-site.js

```diff
@@ -1,7 +1,10 @@
 
 // requires bootbox
+if (typeof msg === 'undefined')
+	var msg = function(msg) {console.error(msg)};
 
-if (typeof msg === 'undefined') var msg = function(msg) {console.error(msg)};
+if (typeof VizorUI === 'undefined')
```

Comment:
for now VizorUI is a grab-bag of calls in which many functions are put, and those may be in different files, before settling.